lib/get-tags-file.coffee

```diff
@@ -15,3 +15,9 @@ module.exports = (directoryPath) ->
 
   tagsFile = path.join(directoryPath, ".TAGS")
   return tagsFile if fs.isFileSync(tagsFile)
+
+  tagsFile = path.join(directoryPath, ".git/tags")
```

Comment:
This won't work on windows because of the hard-coded `/`. Can you change this and the one below to `path.join(directoryPath, ".git", "tags")`?